dot-net-sdk/dto/bandit/BanditEvaluation.cs

```diff
@@ -0,0 +1,35 @@
+using System.Diagnostics.CodeAnalysis;
+
+namespace eppo_sdk.dto.bandit;
+
+public record BanditEvaluation{
+
+    [SetsRequiredMembers]
+    public BanditEvaluation(string flagKey,
```

Comment:
Note we also decided to track and log the optimality gap -- the difference between highest weighed action and selected action.

See [python example](https://github.com/Eppo-exp/python-sdk/blob/main/eppo_client/bandit.py#L91))

I know it's a pain to chase a moving target 😛

Reply:
no worries!